### PR TITLE
Replaced handler information array triplet (endpoint, handler, authorization) with Api

### PR DESCRIPTION
**Pull request by lulco**
Not sure about the naming, but this should replace former array triplet

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ Updates should follow the [Keep a CHANGELOG](http://keepachangelog.com/) princip
 
 * Update nette libs to version 3.0.0 (BC break)
 * Pretty JSON output in API console - without escaping unicode and slashes
+* Replaced handler information array triplet (endpoint, handler, authorization) with HandlerSettings
```
Comment (tomaj) [inline CHANGELOG.md]: iba uvazujem - ten nazov HandlerSettings - keby sme to volali len ze "Api" ? :-) alebo nejaka ina idea ako HandlerSettings?

Comment (lulco) [reply CHANGELOG.md]: :) sak aj ja som napisal ze s tym nazvom nie som uplne stotozneny

**Comment by tomaj** [comment]
Please add UPGRADE.md :-)

**Comment by lulco** [comment]
UPGRADE.md will be added at the end of work ;)

**Pull request merged**